README.md

```diff
@@ -1,5 +1,6 @@
-[![CircleCI](https://circleci.com/gh/andela/andela-societies-frontend.svg?style=svg)](https://circleci.com/gh/andela/andela-societies-frontend)
-[![Coverage Status](https://coveralls.io/repos/github/andela/andela-societies-frontend/badge.svg)](https://coveralls.io/github/andela/andela-societies-frontend)
+[![CircleCI](https://circleci.com/gh/andela/andela-societies-frontend/tree/develop-V2.svg?style=svg)](https://circleci.com/gh/andela/andela-societies-frontend/tree/develop-V2)
```

Comment:
@Kachulio1 are we deprecating develop-v2 after the merge, and using develop solely?

Reply:
Yes ...

Reply:
😕

Reply:
@Gidraff I'd suggest we leave the CircleCI badges and Coverage badges as before and remove the develop-v2 changes.

Reply:
@WinstonKamau Okay. Working on it.